gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,9 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED:
+            _LOGGER.debug(" {{ service.async_client_name }} client is successfully constructed.")
```

Comment:
The ruby logs include information about which credentials will be used with the client which should be included here as well.
```
I, [2024-09-30T16:52:15.809067 #1917313]  INFO -- google-cloud-secret_manager-v1: Created client for google.cloud.secretmanager.v1.SecretManagerService -- {"clientId":3220,"system":"google-cloud-secret_manager-v1","serviceName":"google.cloud.secretmanager.v1.SecretManagerService","credentialsId":3260,"credentialsType":"Google::Auth::ServiceAccountCredentials","credentialsScope":["https://www.googleapis.com/auth/cloud-platform"],"useSelfSignedJWT":false,"universeDomain":"googleapis.com"}
```

Reply:
I added logging for `serviceName` via https://github.com/googleapis/gapic-generator-python/pull/2265/commits/a7158546cc47eb8b78174213732733a75c008c38 but it only appears when using structured logging

```
{"timestamp": "2024-11-28 19:30:07,202", "severity": "DEBUG", "name": "google.showcase_v1beta1.services.echo.client", "message": "Created client `google.showcase_v1beta1.EchoClient`.", "serviceName": "google.showcase.v1beta1.Echo"}
```

```
DEBUG:google.showcase_v1beta1.services.echo.client:Created client `google.showcase_v1beta1.EchoClient`.
```

Reply:
Extra fields will only be included in the structured logging format. We can support this as a follow up: https://github.com/googleapis/gapic-generator-python/issues/2268.